infra/runtime/dask/build/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  dask:
+    build: .
+    image: cloudfuse-io/l12n:databend
```

Comment:
⁉️ Is this the proper docker imago to use here?

Reply:
well spotted, thanks! It quite amazing that it worked despite this typo 😄